presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -434,4 +434,36 @@ public Optional<SystemTable> getSystemTable(ConnectorSession session, SchemaTabl
 
         return getIcebergSystemTable(tableName, icebergTable);
     }
+
+    /**
+     * Deletes the files from a particular scan
+     *
+     * @return
+     */
```

Comment:
The comment here looks unnecessary. Could you remove it?

Reply:
I kept the comment but updated it to properly reflect the functionality